apps/frontend/src/components/chat/ChatBubble.tsx

```
@@ -1,102 +1,105 @@
+											className={`mt-2 p-3 rounded ${
+												message.sender === "user"
+													? "bg-blue-700 text-white"
+													: "bg-gray-50 text-gray-800"
+											}`}
+										>
+											<p className="mb-2">"{citation.snippet}"</p>
+											<div className="flex justify-between items-center text-xs">
+												<span>{citation.source}</span>
+												<a
+													href="#"
+													className={`flex items-center ${
```

Biome (1.9.4)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)